src/modules/submitScore.js

```diff
@@ -1,27 +1,23 @@
-const API_URL =  "https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xHcNAgk6UkMi3fEdct8S/scores";
+const API_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xHcNAgk6UkMi3fEdct8S/scores';
+const userName = document.getElementById('userName');
+const userScore = document.getElementById('userScore');
 
-function postMessage(data) {
+class Data {
+  'user':"userName.value"
+
+  'score': "userName.value"
+}
```

Comment:
- [ ] I understand you are trying to test new things, but this is not a valid syntax. I would suggest you keep your code dry and readable. I would suggest that you pass them in your function as parameters/arguments like this:
`postMessage(userName.value, userScore.value)`;

Reply:
Thanks, I will take a path similar to the one you suggest.